doc/source/whatsnew/v0.20.0.txt

```diff
@@ -49,7 +49,7 @@ Other enhancements
 ^^^^^^^^^^^^^^^^^^
 
 - ``pd.read_excel`` now preserves sheet order when using ``sheetname=None`` (:issue:`9930`)
-
+- ``Series`` provides a ``to_excel`` method to output Excel files (:issue:`8825`)
```

Comment:
Series has gained a ``.to_excel`` method, to output Excel files, similar to ``DataFrame.to_excel``